src/library_syscall.js

```diff
@@ -826,7 +829,6 @@ var SyscallsLibrary = {
     path = SYSCALLS.calculateAt(dirfd, path);
     // remove a trailing slash, if one - /a/b/ has basename of '', but
     // we want to create b in the context of this function
-    path = PATH.normalize(path);
     if (path[path.length-1] === '/') path = path.substr(0, path.length-1);
```

Comment:
Should this now be `while` instead of `if` to handle multiple tailing slashes?

Reply:
We can actually just delete this, it works fine to have a trailing slash. `basename()` does:
```js
path = path.replace(/\/$/, "");
```